src/ServiceStack.OrmLite.Oracle/OracleOrmLiteDialectProvider.cs

```diff
@@ -232,14 +247,28 @@ public override string GetQuotedValue(object value, Type fieldType)
                 var dateValue = (DateTime)value;
                 string iso8601Format = "yyyy-MM-dd";
                 string oracleFormat = "YYYY-MM-DD";
-                if (dateValue.ToString("yyyy-MM-dd HH:mm:ss.fff").EndsWith("00:00:00.000") == false)
+                if (dateValue.ToString("yyyy-MM-dd HH:mm:ss.fffffff").EndsWith("00:00:00.0000000") == false)
```

Comment:
The other pattern that might help the readability of the generated SQL would be to test if the value has any fractional seconds (same example from ServiceStack.Text above).

Something like:
```
var hasFractionalSecs = (timeOfDay.Milliseconds != 0) || ((timeOfDay.Ticks%TimeSpan.TicksPerMillisecond) != 0);

if( hasFractionalSecs )
{
    iso8601Format += ".fffffff";
    oracleFormat += ".FF9";
}
```

I don't think this is as important as the start-of-day test, but it certainly will remove a lot of `.000000000` chatter from the debug logs for any times that are human-entered.